public/app/features/org/state/actions.ts

```diff
@@ -52,3 +52,21 @@ export function createOrganization(
     window.location.href = getConfig().appSubUrl + '/org';
   };
 }
+
+export function validateOrganization(
+  orgName: string,
+  dependencies: OrganizationDependencies = { getBackendSrv: getBackendSrv }
+): ThunkResult<any> {
+  return async () => {
+    try {
+      await dependencies.getBackendSrv().get(`api/orgs/name/${encodeURI(orgName)}`);
+    } catch (error) {
+      if (error.status === 404) {
+        error.isHandled = true;
+        return true;
+      }
+      return 'Something went wrong';
+    }
+    return 'Organization already exists';
+  };
+}
```

Comment:
This function doesn't actually connect to redux and dispatch an action, so im not sure if it makes the most sense here.

Additionally, if we are refactoring this, I would like to see the the return typed properly rather than preserving the dangerous `any`

Reply:
yea, I agree, not sure I see the value in making this Thunk

Reply:
yeah...according to the discussions we had on slack, the decision was to disable input validation entirely since we're already doing same on submit.